pandas/_libs/tslibs/__init__.py

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env python
```

Comment:
So that it can be imported python-style.  Or if you're referring to the shebang that's force of habit.